app/uk/gov/hmrc/helptosavereminder/config/Scheduler.scala

```diff
@@ -16,40 +16,22 @@
 
 package uk.gov.hmrc.helptosavereminder.config
 
-import akka.actor.{ActorSystem, Props}
+import akka.actor.ActorSystem
+import play.api.Logging
+import uk.gov.hmrc.helptosavereminder.services.EmailSenderService
 
 import javax.inject.{Inject, Singleton}
-import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
-import uk.gov.hmrc.http.HttpClient
-import play.api.{Configuration, Environment, Logging}
-import uk.gov.hmrc.helptosavereminder.actors.ProcessingSupervisor
-import uk.gov.hmrc.helptosavereminder.connectors.EmailConnector
-import uk.gov.hmrc.helptosavereminder.models.ActorUtils._
-import uk.gov.hmrc.mongo.MongoComponent
-import uk.gov.hmrc.mongo.lock.MongoLockRepository
-
 import scala.concurrent.ExecutionContext
 
 @Singleton
 class Scheduler @Inject() (
```

Comment:
is it possible to rename this so as not to cause confusion with the actor Scheduler?

Reply:
🤔 I thought package names are sufficient to disambiguate between different kinds of classes. It already exists in Scala (ex. mutable.List and List). Adding an Actor suffix to an actor seems redundant when it's already in the `.actor` package.